### PR TITLE
monitoring runbook - add delete shards step

### PR DESCRIPTION
**Pull request by toschneck**
**What this PR does / why we need it**:
Add a potential soltion for the alert `ElasticsearchUnassignedShards`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

this will be end in the documentation

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: config/monitoring/prometheus/rules/src/general/elasticsearch-exporter.yaml [hunk 1]
```diff
@@ -43,3 +43,5 @@ groups:
     runbook:
       steps:
       - Check the shard allocation via `http://es-data:9200/_cat/shards?v`.
+      - potential fix could be to delete the `UNASSIGNED` shards by `curl http://es-data:9200/_cat/shards | grep UNASSIGNED | awk {'print $1'} | xargs -i curl -XDELETE \"http://es-data:9200/{}\"`
```
Comment (xrstf) [inline config/monitoring/prometheus/rules/src/general/elasticsearch-exporter.yaml:46]: This is not a potential fix, this is the "get the big hammer and nail everything into oblivion" solution, similar to simply deleting the Elasticsearch volmes. ;-) So I would rephrase this to

a) Be a full, English sentence with a capital letter at the beginning, proper grammer and a full stop (dot) at the end. And
b) remove the "could be" to "Delete unassigned shards by doing `....`. Note that this means losing data."

**Comment by kdomanski** [comment]
@toschneck please rebase

**Comment by xrstf** [comment]
ES is deprecated, soon to be removed, we never agreed that deleting shards is really a helpful thing to advice, so I think we can close this. Please re-open if you feel otherwise. :)
